src/clj/rems/css/style_utils.clj

```diff
@@ -2,11 +2,30 @@
   "The namespace contains the helpful style utils that are meant to be kept separate
    the main styles file to keep it cleaner."
   (:require [clojure.string :as str]
-            [garden.stylesheet :as stylesheet]
             [rems.config :refer [env]]))
 
 
-;; Customazable theme related functions
+;; Customizable theme related functions
+
+(def ^:private ignore-theme-var-error?
+  ;; We don't want to include all these localized versions in
+  ;; config-defaults.edn. Luckily there are only a few so we can just
+  ;; have a whitelist.
+  #{:logo-name-fi
+    :logo-name-fi-sm
+    :navbar-logo-name-fi
+    :logo-name-sv
+    :logo-name-sv-sm
+    :navbar-logo-name-sv})
+
+(defn- theme-get
```

Comment:
Should we name this / `get-theme-attribute` to `getx-theme-attribute` or `theme-getx`?

Reply:
I like `theme-getx`. Maybe the current `theme-get` should be called `theme-getx-impl` and the public `get-theme-attribute` could be renamed `theme-getx`?

Reply:
Yeah sounds good to me.